--- a/install.rdf	Fri Nov 15 11:04:18 2013
+++ b/install.rdf	Fri Nov 15 11:06:54 2013
@@ -13,7 +13,7 @@
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
         <em:minVersion>19.0</em:minVersion>
-        <em:maxVersion>20.*</em:maxVersion>
+        <em:maxVersion>28.*</em:maxVersion>
       </Description>
     </em:targetApplication>
 
@@ -22,7 +22,17 @@
     <em:description>Translate English webpage into Chinese using Netease Youdao webpage translator 2.0. Provide full-text translation, and isolated words translation based on the level of your English skill.</em:description>
     <em:creator>Mozilla Online Limited</em:creator>
     <em:homepageURL>http://firefox.com.cn</em:homepageURL>
-    
-    
+
+    <em:updateURL>http://g-fox.cn/chinaedition/addons/youdao-translate/update.rdf</em:updateURL>
+    <em:updateKey>MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDXo3R/qV/Q2vBn105LXSXZ9y3/BXkWO42xWtSUSqgCBPjQfroKpY6B7j9imipJFS9nOQ//LePXz8noaRNyi0ZWJITuhFV1u7mwn89NLiUWUS4irJ6SBKve00xNOagLk/ja0a8u9zH57dCfjj2W6iTO4RI0N7zQ1mK7iQbXvtabHQIDAQAB</em:updateKey>
+    <em:localized>
+      <Description>
+        <em:locale>zh-CN</em:locale>
+        <em:name>网页翻译</em:name>
+        <em:description>使用火狐浏览英文网页，帮您一键翻译成中文，阅读无障碍。使用有道网页翻译2.0，您既可以选择全文翻译，也能按照您的英文水平、只翻译相应部分的词汇，阅读理解更方便。</em:description>
+        <em:creator>Mozilla Online Ltd.</em:creator>
+        <em:homepageURL>http://www.firefox.com.cn/</em:homepageURL>
+      </Description>
+    </em:localized>
   </Description>
-</RDF>
\ No newline at end of file
+</RDF>
